docs/examples/DataSet/Pedestrian example of subscribing to a DataSet.ipynb

```diff
@@ -210,112 +200,45 @@
-      "........O.......................\n",
-      "..........O.....................\n",
-      "..........O.....................\n",
-      "............O...................\n",
-      ".............O..................\n",
-      "..............O.................\n",
-      "..............O.................\n",
       "...............O................\n",
+      "................O...............\n",
       "..................O.............\n",
-      "....................O...........\n",
+      "...................O............\n",
       ".....................O..........\n",
       "......................O.........\n",
       "........................O.......\n",
-      ".......................O........\n",
-      "..........................O.....\n",
-      "..........................O.....\n",
-      "...........................O....\n",
-      "..........................O.....\n",
+      ".........................O......\n",
       "...........................O....\n",
       "............................O...\n",
-      "...........................O....\n",
-      ".............................O..\n",
       "..............................O.\n",
-      "............................O...\n",
-      "............................O...\n",
-      ".............................O..\n",
-      ".............................O..\n",
-      "...........................O....\n",
-      "...........................O....\n",
-      "...........................O....\n",
-      ".........................O......\n",
-      "........................O.......\n",
-      ".......................O........\n",
-      ".......................O........\n",
-      "......................O.........\n",
-      ".....................O..........\n",
-      "....................O...........\n",
-      "..................O.............\n",
-      ".................O..............\n"
+      "...............................O\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Exception in thread Thread-7:\n",
+      "Traceback (most recent call last):\n",
+      "  File \"C:\\Users\\jenielse\\AppData\\Local\\Continuum\\miniconda3\\envs\\qcodes\\lib\\threading.py\", line 916, in _bootstrap_inner\n",
+      "    self.run()\n",
+      "  File \"c:\\users\\jenielse\\source\\repos\\qcodes\\qcodes\\dataset\\data_set.py\", line 143, in run\n",
+      "    self._loop()\n",
+      "  File \"c:\\users\\jenielse\\source\\repos\\qcodes\\qcodes\\dataset\\data_set.py\", line 168, in _loop\n",
+      "    self._call_callback_on_queue_data()\n",
+      "  File \"c:\\users\\jenielse\\source\\repos\\qcodes\\qcodes\\dataset\\data_set.py\", line 157, in _call_callback_on_queue_data\n",
+      "    self.callback(result_list, self._data_set_len, self.state)\n",
+      "  File \"<ipython-input-9-348631bac194>\", line 15, in ASCII_plotter_5bit\n",
+      "    plotline[yvalue] = 'O'\n",
+      "IndexError: list assignment index out of range\n",
```

Comment:
exception?

Reply:
This is indeed strange I did not change this logic